File: UnicoVehicle/UnicoVehicle.DTO/Company.cs
using System;
namespace UnicoVehicle.DTO
{
    public class Company
    {
        public int CompanyId { get; set; }
        public Miscellaneous.District District { get; set; }
        public string CompanyName { get; set; }
        public string CompanyHead { get; set; }
    }
}

File: Flutter App/unicoVehicle/lib/Models/Company.dart
import './Miscellaneous/District.dart';

class Company {
  final int? companyId;
  final District? district;
  final String? companyName;
  final String? companyHead;

  Company({
    this.companyHead,
    this.companyId,
    this.companyName,
    this.district,
  });
}
